infra/modules/azure_api_management/variables.tf

```diff
@@ -225,16 +223,75 @@ variable "application_insights" {
 
   validation {
     condition     = var.application_insights.sampling_percentage >= 0 && var.application_insights.sampling_percentage <= 100
-    error_message = "Invalid `sampling_percentage` value provided. Valid values are between 0 and 100"
+    error_message = "Invalid 'sampling_percentage' value provided. Valid values are between 0 and 100"
   }
 
   validation {
     condition     = contains(["verbose", "information", "error"], var.application_insights.verbosity)
-    error_message = "Invalid `verbosity` value provided. Valid values are 'verbose', 'information', 'error'"
+    error_message = "Invalid 'verbosity' value provided. Valid values are 'verbose', 'information', 'error'"
```

Comment:
We usually adopt double quotes with escape `\"`

Reply:
I updated them to follow the convention